source/write-operations/bulk-writes.txt

```diff
@@ -1,58 +1,353 @@
+         fields </reference/command/insert/#command-fields>` guide in the
+         MongoDB Server manual.  
+
+   * - ``comment(String comment)``
+     - | Attaches a ``String`` comment to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         MongoDB Server manual.  
+
+   * - ``let(Bson variables)``
+     - | Specifies a map of parameter names and values. Values must be constant or closed
+         expressions that don't reference document fields. For more information,
+         see the :manual:`let statement
+         </reference/command/update/#std-label-update-let-syntax>` in the
+         MongoDB Server manual.
+
+   * - ``ordered(Boolean ordered)``
+     - | If set to ``True``, the driver sends documents to the
+         server in the order provided. If an error occurs, the driver
+         and server cancel all remaining write operations.
+       | Defaults to ``True``. 
+
+The following example calls the ``bulkWrite()`` method from the preceding
+example but sets the ``ordered`` option to ``False``:
+
+.. io-code-block::
+   :copyable: 
+
+   .. input:: /includes/write/bulk-write.java
+      :start-after: start-bulk-write-unordered
+      :end-before: end-bulk-write-unordered
+      :language: java
+      :dedent:
+
+   .. output::
+
+      AcknowledgedBulkWriteResult{insertedCount=2, matchedCount=2, removedCount=1, modifiedCount=2, upserts=[], inserts=[BulkWriteInsert{index=0, id=BsonObjectId{value=66a7e03cce430c5854b6caf9}}, BulkWriteInsert{index=1, id=BsonObjectId{value=66a7e03cce430c5854b6cafa}}]}
+
+If any of the write operations in an unordered bulk write fail, the {+driver-short+}
+reports the errors only after attempting all operations.
+
+.. note::
+
+   Unordered bulk operations do not guarantee order of execution. The order can
+   differ from the way you list them to optimize the runtime.
+
+Return Value
+------------
+
+The ``bulkWrite()`` method returns a ``BulkWriteResult`` object. The
+``BulkWriteResult`` object contains the following methods:
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - .. code-block:: none
+          :copyable: false
+
+           acknowledged(int insertedCount,
+           int matchedCount,
+           int removedCount, Integer modifiedCount,
+           List<BulkWriteUpsert> upserts,
+           List<BulkWriteInsert> inserts)
+
+     - | Indicates if the server acknowledged the write operation.
+   
+   * - .. code-block:: none
+          :copyable: false
+
+           acknowledged(com.mongodb
+           .internal.bulk
+           .WriteRequest.Type type,
+           int count,
+           Integer modifiedCount,
+           List<BulkWriteUpsert> upserts,
+           List<BulkWriteInsert> inserts)
+
+     - | Indicates if the server acknowledged the write operation.
```

Comment:
These methods do not indicate anything, they create instances of `BulkWriteResult`.

P.S. I think that duplicating the Java API documentation in the driver docs is not only unhelpful, but rather harmful. I also know, that this decision is up to the docs team :)

Reply:
Taking over this PR now that Angela's internship has ended.

Agree that this section feels redundant given that we also link to the `BulkWriteResult` API docs on this page. I'm of the mind to remove this section.